crates/agglayer-aggregator-notifier/src/certifier/mod.rs

```diff
@@ -278,6 +277,12 @@ where
             .await
             .map_err(|_| CertificationError::TrustedSequencerNotFound(network_id))?;
 
+        let prev_pessimistic_root = self
+            .l1_rpc
+            .get_prev_pessimistic_root(*network_id)
+            .await
+            .map_err(|_| CertificationError::LastPessimisticRootNotFound(network_id))?;
+
```

Comment:
Maybe we should have a method that returns both params, instead of doing 2 RPC calls

Reply:
yeah, hard to name it tho, maybe we could have one method which return the entire rollup data structure which is fetched entirely anyway

Reply:
ah not exactly the same actually, in one case we ask for one `RollupDataReturn` and in the other case the `RollupDataReturnV2`, so not entirely sure we can make single call there